Updates for managed cluster labels variable (#4924)
open-cluster-management/rhacm-docs: governance/template_functions.adoc

[#template-functions]
= Template functions

> Template functions, such as resource-specific and generic `lookup` template functions, are available for referencing Kubernetes resources on the hub cluster (using the `{{hub ... hub}}` delimiters), or managed cluster (using the `{{ ... }}` delimiters). See _Template processing_ for more details. The resource-specific functions are used for convenience and makes content of the resources more accessible. If you use the generic function, `lookup`, which is more advanced, it is best to be familiar with the YAML structure of the resource that is being looked up. In addition to these functions, utility functions such as `base64encode`, `base64decode`, `indent`, `autoindent`, `toInt`, `toBool`, and more are also available.

Reviewer: ```suggestion
Template functions, such as resource-specific and generic `lookup` template functions, are available for referencing Kubernetes resources on the hub cluster (using the `{{hub ... hub}}` delimiters), or on the managed cluster (using the `{{ ... }}` delimiters). See _Template processing_ for more details. The resource-specific functions are used for convenience and makes content of the resources more accessible. If you use the generic function, `lookup`, which is more advanced, familiarize yourself with the YAML structure of the resource that is being looked up. In addition to these functions, utility functions such as `base64encode`, `base64decode`, `indent`, `autoindent`, `toInt`, `toBool`, and more are also available.
```

Author: Great suggestions! TEAM ACTIVE TONE !
